ca_postalcodes.csv

```diff
@@ -1158,10 +1172,10 @@ C1C,Charlottetown North,Prince Edward Island,46.2688,-63.1097,-4,1
 C1E,Charlottetown West,Prince Edward Island,46.2607,-63.16,-4,1
 C1N,Summerside,Prince Edward Island,46.3907,-63.7868,-4,1
 G0A,Capitale-Nationale (Stoneham),Quebec,46.8524,-72.0259,-5,1
-G0B,Cap-aux-Meules,Quebec,47.3983,-61.7742,-5,1
+G0B,Cap-aux-Meules,Quebec,47.3983,-61.7742,-4,1
 G0C,Gaspésie-Sud (New Richmond),Quebec,48.1496,-65.7053,-5,1
 G0E,Gaspésie-Nord (Grande-Vallée),Quebec,48.9298,-64.3438,-5,1
-G0G,Côte-Nord/Anticosti (Schefferville),Quebec,50.1169,-62.4136,-5,1
+G0G,Côte-Nord/Anticosti (Schefferville),Quebec,50.1169,-62.4136,-4,0
```

Comment:
This is wrong because the coordinate is in the water because this prefix is for East Quebec and Anticosti island. it should be -5.1 like everything else.